### ISSUE TITLE
Data Visualizer Base Class

### ISSUE THREAD

**Issue by bbengfort** [DistrictDataLabs/yellowbrick#63]
The `fit` method is identical in both `RadViz` and `ParallelCoordinates` - this method should be given to a higher level class, and `RadViz` and `ParallelCoordinates` should extend it. 

I'm thinking we create an intermediate class between `FeatureVisualizer` called `DataVisualizer` just in case there are further abstractions down the road.

**Comment by bbengfort** [comment]
See #80 and #18 for feature visualizers that are slightly different.

**Comment by bbengfort** [comment]
Refactoring complete and tested. This paves the way for #73 since the changes will be applied to the `DataVisualizer` class and help with both RadViz and ParallelCoords. 

A couple of notes on this:

- I made sure the `__init__` of `DataVisualizer`, `ParallelCoordinates`, and `RadViz` were all identical. 
- I ensured that all the above classes had similar docstrings for documentation purposes. 
- The move of `fit()` to `features/base.py` may make it tougher to code `DataVisualizer` classes, but also solves a lot of problems.
